### DISCUSSION TITLE
Maximum Number of Turbines/Farm Size?

### DISCUSSION THREAD

**Discussion by rhester5** [NatLabRockies/floris#212] [Q&A] [answered]
Hi! I'm working on a project to compare FLORIS with other modeling tools for wind farms. Is there a known number of turbines and/or a farm size at which point the results from FLORIS are no longer reliable?

**Comment by paulf81** [comment, marked as answer]
Not that we've observed yet, we've run cases with several hundreds of turbines for example, do you need to go much larger than that?

**Reply by rhester5** [reply]
No, that should definitely be sufficient. Thanks!
